File: fip_collab/strain_mks/matthew_250FZreduced/view_orientations.py
# -*- coding: utf-8 -*-
"""
Created on Fri May 23 14:25:50 2014

This script evaluates the success of a given MKS calibration and validation
through metrics like MASE and maximum error as well as plotting strain
fields and histograms.

@author: nhpnp3
"""

import numpy as np
import matplotlib.pyplot as plt
import scipy.io as sio
from mpl_toolkits.mplot3d import Axes3D


#euler = sio.loadmat('euler_250cal.mat')['euler']

euler = np.load('euler_50val.npy')

sn=25
max = 500

fig = plt.figure()
ax3D = fig.add_subplot(111, projection='3d')
p3d = ax3D.scatter(euler[:max,sn,0],euler[:max,sn,1],euler[:max,sn,2])
plt.set_aspect('equal')

File: fip_collab/strain_mks/matthew_250FZreduced/view_orientations.m
load euler_250cal.mat

st = 1;
en = 500;
sn = 1;

close all

figure(1)
scatter3(euler(st:en,sn,1),euler(st:en,sn,2),euler(st:en,sn,3),'kx')
axis equal

figure(2)
scatter(euler(st:en,sn,1),euler(st:en,sn,2),'kx')
axis equal
